### ISSUE TITLE
GPU-related issue after switching from an empty desktop in Chrome beta, dev

### ISSUE THREAD

**Issue by rsmith31415** [baskerville/bspwm#1244]
I noticed after updating some packages a very unusual issue in which nodes for google chrome beta (`Google Chrome 88.0.4324.50 beta`)  and google chrome dev (`Google Chrome 89.0.4356.6 dev`) are not visible after switching from an empty desktop: 

![screencast-chrome-beta](https://user-images.githubusercontent.com/599274/103333019-eaea2b00-4a31-11eb-8d94-2cf30c6353a2.gif)

It is only after moving the mouse over the "empty" space that chrome becomes visible again. The behavior is quite easy to replicate by launching any floating node in the same desktop, although even without a floating node, there is a small delay after moving between desktops. It is possible to "fix" it by adding the flag `--disable-gpu` as you can see in the second part of the video. At the moment, neither google chrome stable nor chromium (`Google Chrome 87.0.4280.88`, `Chromium 87.0.4280.66`) have this issue.  

I initially thought it was related to this other issue (https://github.com/baskerville/bspwm/issues/745). Unfortunately, the proposed solutions (e.g. `unredir-if-possible = false;`) are not working in this case. The issue can also be reproduced using a minimal `bspwmrc` configuration file. 

The relevant information can be seen below. If you need the `xtrace` output, please let me know. Any help would be appreciated.

-----
➜  ~  bspwm -v
0.9.10-17-g7012d99
➜  ~  lsb_release -a
... Ubuntu 18.04.5 LTS
➜  ~  nvidia-smi
Tue Dec 29 23:20:20 2020
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 390.138                Driver Version: 390.138                   |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  GeForce GTX 105...  Off  | 00000000:01:00.0 Off |                  N/A |
| N/A   54C    P8    N/A /  N/A |    114MiB /  4040MiB |      9%      Default |
+-------------------------------+----------------------+----------------------+

+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|    0      1631      G   /usr/lib/xorg/Xorg                           103MiB |
|    0      2640      G   alacritty                                      8MiB |
+-----------------------------------------------------------------------------+

.config/bspwm/bspwmrc

```
#! /bin/sh

sxhkd &
picom &

echo 3300 | sudo tee /sys/class/backlight/intel_backlight/brightness
redshift -O 2800

xsetroot -cursor_name left_ptr
setxkbmap -layout es,us -option numpad:microsoft

bspc config border_width        1
bspc config window_gap          0

bspc config top_padding 2

bspc config split_ratio          0.50
bspc config borderless_monocle   true
bspc config gapless_monocle      true

bspc config external_rules_command /home/user/.config/bspwm/external_rules

bspc config focused_border_color "#959595"
bspc config click_to_focus true

bspc monitor -d I II III IV V VI VII VIII IX X

$HOME/.config/polybar/launch.sh

wmname LG3D

feh --bg-scale ~/Pictures/wallpaper.jpeg

nm-applet &
dropbox start &
clipit &

bspc config automatic_scheme spiral
bspc config initial_polarity second_child

bspc config ignore_ewmh_focus true
```

**Comment by rsmith31415** [comment]
A small update: 

Unfortunately, the `--disable-gpu` flag doesn't really work. The same delay is noticeable when switching from a desktop running other apps to the desktop running one of the affected versions of Chrome, so the issue is not restricted to moving from empty desktops as I initially thought. 

I also found that chrome beta 88.0.4324.27 (https://chromereleases.googleblog.com/2020/12/beta-channel-update-for-desktop.html) is the first release with this issue (and `87.0.4280.66` is the previous release without this flaw). The changelog can be seen [here](https://chromium.googlesource.com/chromium/src/+log/87.0.4280.66..88.0.4324.27/?n=450), although looking at the commits I wasn't able to identify the cause of this issue. 

By the way, old versions of chrome beta can be downloaded via `https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-beta/google-chrome-beta_$VERSION-1_amd64.deb`, just in case that's needed.

**Comment by rsmith31415** [comment]
Now this behavior can be replicated in Chrome stable (88.0.4324.150).  For some reason, Chrome unstable (90.0.4408.0) looks okay right now, so it is possible this issue could be resolved on its own. Any comments or feedback?

**Comment by MonstrousOgre** [comment]
I have the same problem with Vivaldi stable. It could be an issue with Chromium, but it never happened when I used it on any other WM with multiple desktops.

It doesn't happen all the time for me though. I'm not sure what exactly is causing it but I can sometimes switch back to the desktop and it appears normally.

**Comment by rsmith31415** [comment]
Could it be that this is a bug in Chromium? I just noticed the same issue in Qtile.

**Comment by rsmith31415** [comment]
I think this is getting worse: 

![screencast-chrome-stable](https://user-images.githubusercontent.com/599274/108175814-ce2fb280-70c6-11eb-9b76-f8282758d06c.gif)

In the screencast, I'm launching a new incognito window in the latest version of chrome stable (88.0.4324.182), moving that window to the third workspace and then going back to the second workspace.

There are a couple of recent comments in the following bug report: https://bugs.chromium.org/p/chromium/issues/detail?id=370022#c106, in that case, for awesomeWM and i3.

**Comment by AndrewVos** [comment]
This seems to be fixed on `89.0.4389.72-1`.
